envoy/datadog_checks/envoy/metrics.py

```diff
@@ -322,6 +322,22 @@
     'envoy_server_initialization_time_ms': 'server.initialization_time_ms',
     'envoy_workers_watchdog_miss': 'workers.watchdog_miss',
     'envoy_workers_watchdog_mega_miss': 'workers.watchdog_mega_miss',
+    'envoy_cluster_outlier_detection_ejections_enforced_total': 'cluster.outlier_detection.ejections_enforced_total',
+    'envoy_cluster_outlier_detection_ejections_active': 'cluster.outlier_detection.ejections_active',
+    'envoy_cluster_outlier_detection_ejections_overflow': 'cluster.outlier_detection.ejections_overflow',
+    'envoy_cluster_outlier_detection_ejections_enforced_consecutive_5xx': 'cluster.outlier_detection.ejections_enforced_consecutive_5xx',
+    'envoy_cluster_outlier_detection_ejections_enforced_success_rate': 'cluster.outlier_detection.ejections_enforced_success_rate',
+    'envoy_cluster_outlier_detection_ejections_detected_success_rate': 'cluster.outlier_detection.ejections_detected_success_rate',
+    'envoy_cluster_outlier_detection_ejections_enforced_consecutive_gateway_failure': 'cluster.outlier_detection.ejections_enforced_consecutive_gateway_failure',
+    'envoy_cluster_outlier_detection_ejections_detected_consecutive_gateway_failure': 'cluster.outlier_detection.ejections_detected_consecutive_gateway_failure',
+    'envoy_cluster_outlier_detection_ejections_enforced_consecutive_local_origin_failure': 'cluster.outlier_detection.ejections_enforced_consecutive_local_origin_failure',
+    'envoy_cluster_outlier_detection_ejections_detected_consecutive_local_origin_failure': 'cluster.outlier_detection.ejections_detected_consecutive_local_origin_failure',
+    'envoy_cluster_outlier_detection_ejections_enforced_local_origin_success_rate': 'cluster.outlier_detection.ejections_enforced_local_origin_success_rate',
+    'envoy_cluster_outlier_detection_ejections_detected_local_origin_success_rate': 'cluster.outlier_detection.ejections_detected_local_origin_success_rate',
+    'envoy_cluster_outlier_detection_ejections_enforced_failure_percentage': 'cluster.outlier_detection.ejections_enforced_failure_percentage',
+    'envoy_cluster_outlier_detection_ejections_detected_failure_percentage': 'cluster.outlier_detection.ejections_detected_failure_percentage',
+    'envoy_cluster_outlier_detection_ejections_enforced_failure_percentage_local_origin': 'cluster.outlier_detection.ejections_enforced_failure_percentage_local_origin',
+    'envoy_cluster_outlier_detection_ejections_detected_failure_percentage_local_origin': 'cluster.outlier_detection.ejections_detected_failure_percentage_local_origin',
```

Comment:
Taking from https://github.com/DataDog/integrations-core/blob/bf691a462c83897ae30930647d48db991b36340b/envoy/datadog_checks/envoy/metrics.py#L3049-L3184

Reply:
I think it is a linter error. How should I fix it?

https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=96747&view=logs&j=910f82a4-622d-5ace-f877-8b75c045486c&t=12901f7b-620f-5414-43d1-3ae27e5163d7&l=620

```
./datadog_checks/envoy/metrics.py:328:121: E501 line too long (137 > 120 characters)
./datadog_checks/envoy/metrics.py:329:121: E501 line too long (131 > 120 characters)
./datadog_checks/envoy/metrics.py:330:121: E501 line too long (131 > 120 characters)
./datadog_checks/envoy/metrics.py:331:121: E501 line too long (161 > 120 characters)
./datadog_checks/envoy/metrics.py:332:121: E501 line too long (161 > 120 characters)
./datadog_checks/envoy/metrics.py:333:121: E501 line too long (171 > 120 characters)
./datadog_checks/envoy/metrics.py:334:121: E501 line too long (171 > 120 characters)
./datadog_checks/envoy/metrics.py:335:121: E501 line too long (157 > 120 characters)
./datadog_checks/envoy/metrics.py:336:121: E501 line too long (157 > 120 characters)
./datadog_checks/envoy/metrics.py:337:121: E501 line too long (143 > 120 characters)
./datadog_checks/envoy/metrics.py:338:121: E501 line too long (143 > 120 characters)
./datadog_checks/envoy/metrics.py:339:121: E501 line too long (169 > 120 characters)
./datadog_checks/envoy/metrics.py:340:121: E501 line too long (169 > 120 characters)
```